packages/api/endpoints/granules.js

```diff
@@ -165,7 +167,7 @@ const putGranule = async (req, res) => {
     await granulePgModel.get(knex, {
       granule_id: apiGranule.granuleId,
       collection_cumulus_id: pgCollection.cumulus_id,
-    });
+    }); // TODO this should do a select count, not a full record get
```

Comment:
This is an arguable optimization change that might be out of scope, but (assuming knex *does*) there's not a good reason to pull back the full record here.

Reply:
In thinking about this, I think it's a scope concern for this ticket, but I'm going to add a comment for discussion in the Arch meeting as this solidly falls under PO.DAAC's scaling/throughput concerns, GHRC's DB performance and a bunch of other tickets.

Reply:
Added to arch doc, resolving.